scripts/rollup/results.json

```diff
@@ -4,8 +4,8 @@
       "filename": "react.development.js",
       "bundleType": "UMD_DEV",
       "packageName": "react",
-      "size": 55220,
```

Comment:
This of course is unrelated to this change, but we need to record them once in a while anyway. I can remove from this PR though.